tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1221,6 +1221,101 @@ some text
 
 `;
 
+exports[`inline2.php 1`] = `
+<?php
+// test
+?>
+<h1>Test</h1>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+// test
+?>
+<h1>Test</h1>
+
+`;
+
+exports[`inline3.php 1`] = `
+<h1>Test</h1>
+<?php
+// test
+?>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<h1>Test</h1>
+<?php
+// test
+?>
+`;
+
+exports[`inline4.php 1`] = `
+<?php
+/**
+ * A
+ */
+
+// B
+
+/**
+ * C
+ */
+?>
+<h1>Test</h1>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+/**
+ * A
+ */
+// B
+/**
+ * C
+ */
+?>
+<h1>Test</h1>
+
+`;
+
+exports[`inline5.php 1`] = `
+<h1>Test</h1>
+<?php
+// test
+?>
+<h1>Test</h1>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<h1>Test</h1>
+<?php  ?><?php
+// test
+?>
+<h1>Test</h1>
+
+`;
+
+exports[`inline6.php 1`] = `
+<h1>Test</h1>
+<?php
+
+/**
+ * test
+ */
+
+?>
+<h1>Test</h1>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<h1>Test</h1>
+<?php  ?><?php
+/**
+ * test
+ */
+?>
+<h1>Test</h1>
+
+`;
+
+exports[`inline7.php 1`] = `
+<?php echo 1; // comment ?>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php echo 1; ?> // comment
```

Comment:
@evilebottnawi Good point, this is also currently breaking

Reply:
@evilebottnawi Should we handle this issue in a separate PR? It's not related to `inline` logic.

Reply:
@czosel :+1: Just create issue